sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/odbc.yaml

```diff
@@ -183,8 +183,26 @@ provider:
         post:
           # Avoid matching values found on SourceGraph that look like dummy passwords or insertions.
           patternNot:
-            - "^(?i)pass(word|wd)?(?-i)$|^(?i)pwd(?-i)$|^\\[?PLACEHOLDER\\]?$|^%s$|^\\*{3,}$|^\\{\\d*\\}$|^\\#\\{|^\\{{2,}"
+            - "^(?i)pass(word|wd)$"
+            - "^(?i)pwd"
+            - "^\\[?PLACEHOLDER\\]?$"
+            - "^%s$"
+            - "^\\*{3,}$"
+            - "^\\{\\d*\\}$"
```

Comment:
This can be removed if you add the new regexp:
```\\{+[^}]*\\}++```

Reply:
```suggestion
```